debian/control

```diff
@@ -29,7 +29,7 @@ Build-Depends: debhelper (>= 10),
                libboost-dev,
                libspdk-dev,
                libipmctl-dev,
-               libraft-dev (= 0.7.0-1377.g4faff0b),
+               libraft-dev (= 0.7.1-1378.g264a878),
```

Comment:
It would be good to start tagging releases of raft so that we are not using git-hashed versions.

Reply:
TBH, I'd rather us not making raft RPMs at all; just include raft submodule in daos source tarballs.

Reply:
The release process that we use in GitHub does not do this.  It does not have the ability to include submodules.